Example/MultipleInstances-Example/CodeExample/CodeExampleViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  CodeExampleViewController.swift
+//  Example
+//
+//  Created by Hans Seiffert on 14/04/17.
+//  Copyright (c) 2017 Folio Reader. All rights reserved.
+//
+
+import UIKit
+import FolioReaderKit
+
+enum Epub: Int {
+    case bookOne = 0
+    case bookTwo
+
+    var name: String {
+        switch self {
+        case .bookOne:      return "The Silver Chair" // standard eBook
+        case .bookTwo:      return "The Adventures Of Sherlock Holmes - Adventure I" // audio-eBook
+        }
+    }
+
+    var readerIdentifier: String {
+        switch self {
+        case .bookOne:      return "READER_ONE"
+        case .bookTwo:      return "READER_TWO"
+        }
+    }
+
+    var shouldHideNavigationOnTap: Bool {
+        switch self {
+        case .bookOne:      return false
+        case .bookTwo:      return true
+        }
+    }
+
+    var scrollDirection: FolioReaderScrollDirection {
+        switch self {
+        case .bookOne:      return .vertical
+        case .bookTwo:      return .horizontal
+        }
+    }
+
+    var bookPath: String? {
+        return Bundle.main.path(forResource: self.name, ofType: "epub")
+    }
+
+    func retain(folioReaderContainer: FolioReaderContainer) {
+        let appDelegate = (UIApplication.shared.delegate as? AppDelegate)
+
+        switch self {
+        case .bookOne: appDelegate?.epubReaderOne = folioReaderContainer
+        case .bookTwo: appDelegate?.epubReaderTwo = folioReaderContainer
+        }
+    }
+}
+
+class CodeExampleViewController: UIViewController {
+
+    @IBOutlet var bookOne   : UIButton?
+    @IBOutlet var bookTwo   : UIButton?
```

Comment:
As they are declared as `@IBOutlet` the connection/link in the storyboard to the actual variable might not be set which would lead to a crash.

Plus I receive xcode building errors if I even try to do so.
Errors:
```
- @IBOutlet property has non-optional type 'UIButton'
- Class 'ViewController' has no initializers
```